ros_compatibility/src/ros_compatibility/ros_compatible_node.py

```diff
@@ -93,11 +87,13 @@ def shutdown(self):
     import rclpy
     from builtin_interfaces.msg import Time
 
+    latch_on = QoSDurabilityPolicy.RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL
+
     def ros_timestamp(sec=0, nsec=0, from_sec=False):
         time = Time()
         if from_sec:
             time.sec = int(sec)
-            time.nanosec = int((sec - int(sec)) * 1000_000_000)
+            time.nanosec = int((sec - int(sec)) * 1000000000)
```

Comment:
Large numbers with a lot of zeros look scary, in Python you can have 1000_000_000 for legibility

Reply:
i had issues compiling it with the underscores

Reply:
You're right, it seemed to be added just in python 3.6